frontend/src/types.ts

```diff
@@ -1142,7 +1148,9 @@ export interface InsightModel extends Cacheable {
     created_at: string
     created_by: UserBasicType | null
     is_sample: boolean
+    // deprecated: use dashboard_tiles instead
```

Comment:
For some IDE highlighting magic:
```suggestion
     /** @deprecated Use `dashboard_tiles instead */
```